### DISCUSSION TITLE
How to show connected/disconnected and controllable/not controllable

### DISCUSSION THREAD

**Discussion by Andre0512** [Andre0512/hon#44] [Ideas]
The connection/remote control attributes are not displayed consistently at the moment and should be reworked, so here are my thoughts.

We have three status:
1. Appliance disconnected
2. Appliance connected but not controllable
3. Appliance connected and controllable

The suggestion in #28 was to show the controls (start/pause) as unavailable when remote control is disabled. This should work since `v0.7.x`, do you like this solution?

To show if the appliance is connected or not I have these ideas:
1. Show all entities as unavailable (the sensors can't be updated at all while disconnected but then we have some appliances for 95% of time unavailable which is also be a bit ugly)
2. Show "Disconnected" in "Machine Status"
3. Use a binary sensor to show connected/disconnected

Which solution do you like the most? Or do you have another suggestion?

**Comment by finalbillybong** [comment]
Personally I do not like things showing as unavailable on my dashboard. My preference would be an additional sensor Controllable: True/False to go with Remote Control: Connected/Disconnected

Disconnected for Machine Status would also work well I think.

**Comment by zebu14** [comment]
Hello,
Suggestion #28 to show the control start/pause as unavailable when disconnected, seems to be a good idea.

Then, for the connected/disconnected status, both 2nd and 3rd solutions are good to me, with a little preference for the 2nd.
